perception.tax.line: The partner_id field is no more a related

ssaid opened the pull request.

ssaid committed:
[MOD] The partner_id field is no more a related

sebastiken commented:
@ssaid Faltaría ver la parte de código que tiene que escribir el partner, ya que no es más un **related** con *invoice_id*, me parece.

ssaid committed:
[MOD] Set the partner_id of perceptions (default & onchange)

1. When creating new perception, the default partner_id is the partner
of the invoice
2. If the invoice partner is modified set this partner for the
perceptions

ssaid commented:
@sebastiken Controlamos por default y onchange

sebastiken commented:
Gracias @ssaid
